src/applications/personalization/profile/reducers/vaProfile.js

```diff
@@ -73,8 +73,14 @@ function vaProfile(state = initialState, action) {
       return set('cnpPaymentInformationUiState.responseError', null, newState);
     }
 
-    case CNP_PAYMENT_INFORMATION_SAVE_STARTED:
-      return set('cnpPaymentInformationUiState.isSaving', true, state);
+    case CNP_PAYMENT_INFORMATION_SAVE_STARTED: {
+      const cnpPaymentInformationUiState = {
+        ...state.cnpPaymentInformationUiState,
+        responseError: null,
```

Comment:
In writing my tests I discovered this bug, where we did not clear existing save errors when starting a new save operation.